pages/blog/migrating-from-rest.mdx

```diff
@@ -0,0 +1,133 @@
+import withArticle from '../../ui/blog/article';
+
+export const meta = {
+  title: 'Migrating from Schemaless REST API to GraphQL',
+  tags: ['graphql', 'rest', 'graphql-mesh'],
+  author: 'arda',
+  date: '2020-05-31',
+  description:
+    "It is really easy to migrate to GraphQL from a REST API without writing everything from scratch using GraphQL Mesh even you don't have any schema for your existing REST API.",
+  image:
+    '/blog-assets/migrating-from-rest/cover.png',
+};
+
+export default withArticle({ ...meta });
+
+Migrating from REST API to GraphQL may look trivial if you want to avoid rewriting your backend from scratch because your existing REST API might not have a decent schema even if you decide to use GraphQL as a proxy. In that case you still need to write a full schema for yout all endpoints.
```

Comment:
GraphQL was originally created in order to easily provide a powerful API on top of existing code.
The current approach people are using today while migrating from REST API to GraphQL is to create a new schema and use GraphQL as a proxy.
That has a lot of benefits because it gives us an opportunity to rethink the API and improve it, without changing the underlining services.

Let's first start with looking at that approach: